core/gui/src/app/common/service/notification/notification.service.ts

```diff
@@ -8,7 +9,18 @@ import { NzMessageDataOptions, NzMessageService } from "ng-zorro-antd/message";
   providedIn: "root",
 })
 export class NotificationService {
-  constructor(private message: NzMessageService) {}
+  constructor(
+    private message: NzMessageService,
+    private notification: NzNotificationService
+  ) {}
+
+  blank(title: string, content: string, options: NzMessageDataOptions = {}): void {
+    this.notification.blank(title, content, options);
+  }
+
+  remove(id?: string): void {
```

Comment:
seems that this ID is not used. Can you remove it?

Reply:
I am not sure what is not used, the parameter "id?"